stats/src/main/java/io/airlift/stats/QuantileDigest.java

```diff
@@ -371,6 +376,91 @@ public boolean process(Node node)
         return builder.build();
     }
 
+    public double getCDF(long x) {
+        final ImmutableList.Builder<Double> builder = ImmutableList.builder();
```

Comment:
Is this being used as a way to return a single value from the callback traversal? It'd be cleaner to either:
1) Introduce a mutable value holder that the callback sets.
2) Pull the callback code into a named class, add a member variable and accessor.